puppet/kandra/files/memcached_exporter

```diff
@@ -382,12 +382,14 @@ class MemcachedCollector(Collector):
 
         raw_stats = client.stats("sizes")
         sizes_stats = next(iter(raw_stats.values()))
-        if sizes_stats.get("sizes_status") == b"disabled" or sizes_stats == {}:
-            return
-        sizes = sorted([int(x) for x in sizes_stats])
-        yield gauge(
-            "item_max_bytes", "Largest item (rounded to 32 bytes) in bytes.", value=sizes[-1]
-        )
+        if sizes_stats.get("sizes_status") != b"disabled" and sizes_stats != {}:
+            sizes = sorted([int(x) for x in sizes_stats])
+            yield gauge(
+                "item_max_bytes", "Largest item (rounded to 32 bytes) in bytes.", value=sizes[-1]
+            )
+            print("disconnecting")
```

Comment:
Should this print statement be outside the if block?

Reply:
Oops no, it should be deleted. Fixed.